README.md

```diff
@@ -97,11 +97,6 @@ This script does the following steps for you:
 - create a TAB (tock application bundle)
 - if you have a J-Link compatible board connected: flash this TAB to your board (using tockloader)
 
-Instead of specifying an environment variable each time you can permanently configure your platform by writing its name into a file named `platform`, e.g.
-
-```bash
-echo nrf52 > platform
-```
```

Comment:
So, this feature is no longer supported? Then, it should be removed from the `build.rs` script. Or you might consider bringing it back?

Reply:
What do you mean?

Reply:
The `build.rs` checks whether the `PLATFORM` env variable is set and, if not, checks whether the file `platform` is present.

Reply:
It will still do that, I haven't change `build.rs`. I just don't think this needs to be documented in the top level `README` anymore as it isn't really user exposed.